File: src/nl/rubensten/texifyidea/TeXception.java
package nl.rubensten.texifyidea;

/**
 * Exception that is thrown by problems within TeXiFy-IDEA.
 *
 * @author Ruben Schellekens
 */
public class TeXception extends RuntimeException {

    public TeXception() {
        super();
    }

    public TeXception(String message) {
        super(message);
    }

    public TeXception(String message, Throwable cause) {
        super(message, cause);
    }

    public TeXception(Throwable cause) {
        super(cause);
    }

}

File: src/nl/rubensten/texifyidea/TeXception.kt
package nl.rubensten.texifyidea

/**
 * Exception that is thrown by problems within TeXiFy-IDEA.
 *
 * @author Ruben Schellekens
 */
class TeXception : RuntimeException {

    constructor() : super()
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    constructor(cause: Throwable) : super(cause)
}